### DISCUSSION TITLE
IntelliSense en Visual Studio Ide

### DISCUSSION THREAD

**Discussion by Sebas1968** [tailwindlabs/tailwindcss#8048] [Help]
Hola, como puedo tener IntelliSense en visual studio ide? o tener el css completo para cargarlo?

**Comment by CFarhad** [comment]
hi. what do you mean ? visual studio or visual studio code. 
Tailwindcss IntelliSense is not available in visual studio but u can use it in [visual studio code](https://tailwindcss.com/docs/editor-setup)

**Comment by Sebas1968** [comment]
Utilizo visual studio comunity, no es visula studio code
